sdcm/utils/sstable/load_utils.py

```diff
@@ -121,9 +122,8 @@ def run_load_and_stream(cls, node,  # pylint: disable=too-many-arguments
 
             # `load_and_stream` parameter is not supported by nodetool yet. This is workaround
             # https://github.com/scylladb/scylla-tools-java/issues/253
-            load_api_cmd = 'curl -X POST --header "Content-Type: application/json" --header ' \
-                           f'"Accept: application/json" "http://127.0.0.1:10000/storage_service/sstables/{keyspace_name}?' \
-                           f'cf={table_name}&load_and_stream=true"'
+            path = f'/storage_service/sstables/{keyspace_name}?cf={table_name}&load_and_stream=true'
+            load_api_cmd = build_node_api_command(path_url=path, request_method=RequestMethods.POST)
```

Comment:
Before the curl command here was without `-s` flag. Now it will have.
Is it intentional?

Reply:
`-s` is silent mode..... perhaps this function will need it, but @juliayakovlev may know that better...
in any case, i could perhaps remove the `-s` from all occurances, as we could "afford" to have more details in the logs...
WDYT?

Reply:
@fgelcer I don't know how useful the output is. I would like to see opinion from @juliayakovlev